Subscribers/Article.php

```diff
@@ -148,7 +148,7 @@ public function generateMainVariantChange($detailId)
             return;
         }
         // Check if entity is a connect product
-        if (!$attribute->getExportStatus() || $attribute->getExportStatus() == Attribute::STATUS_DELETE) {
+        if (!$this->getHelper()->isProductExported($attribute)) {
```

Comment:
@sbofirov If we have ``Attribute::STATUS_ERROR_PRICE`` isProductExported() will return false. Is it possible for the product to have been exported and then on Update the price fails and the status is set to STATUS_ERROR_PRICE and is this considered a connect product in this case ?